canonical/snapcraft.io
# Fix redirect

## Done
- Fix redirect to login-beta for `/admin` routes
## How to QA
-Go to /login
- Go to any admin route and ensure that it redirects you to authenticate again
    - https://snapcraft-io-4362.demos.haus/admin/model/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
    - https://snapcraft-io-4362.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/signing-keys
- Ensure that you are redirected back to the previous page
## Issue / Card
Fixes #

## Screenshots
